sha3 v0.11.0-rc.8 fails to build on Rust 1.91.1

Building `sha3-0.11.0-rc.8` fails with the following error:

```
   Compiling sha3 v0.11.0-rc.8
error[E0432]: unresolved import `digest::block_buffer::BlockSizes`
  --> /Users/aubs/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/sha3-0.11.0-rc.8/src/block_api.rs:10:5
   |
10 |     block_buffer::BlockSizes,
   |     ^^^^^^^^^^^^^^^^^^^^^^^^ no `BlockSizes` in the root

For more information about this error, try `rustc --explain E0432`.
```

I tried building for `aarch64-apple-darwin` and `wasm32-unknown-unknown` and got this error - I'm not sure if it shows up on different targets.

--- a/sha3/Cargo.toml
+++ b/sha3/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "sha3"
-version = "0.11.0-rc.8"
+version = "0.11.0-rc.9"
 description = """
 Pure Rust implementation of SHA-3, a family of Keccak-based hash functions
 including the SHAKE family of eXtendable-Output Functions (XOFs), as well as

